src/ldp/representation/RepresentationMetadata.ts

```diff
@@ -12,9 +12,9 @@ export interface RepresentationMetadata {
    */
   raw: Quad[];
   /**
-   * The metadata profiles.
+   * Optional metadata profiles.
    */
-  profiles: string[];
+  profiles?: string[];
```

Comment:
What is the  difference between an empty array and missing profiles?

Reply:
Not that much, but it allows us not to put `profiles: []` anywhere in the tests as proposed here https://github.com/solid/community-server/pull/52#discussion_r470643241 
Do you want me to change anything though?

Reply:
Right; I wouldn't change the typings just for the test though. Is that the case, @joachimvh?

Reply:
It's something I have been wanting to do for a while (following the discussion [here](https://github.com/solid/community-server/issues/18#issuecomment-663476245)), but never did a PR for specifically, so this seemed as good as a place as any to finally change it to me.

Reply:
OK I'm good.